packages/system/kairos-agent/collection.yaml

```diff
@@ -12,7 +12,7 @@ packages:
     description: "Lifecycle agent for kairos"
   - name: "kairos-agent"
     category: "fips"
-    version: "2.10.2"
+    version: "2.10.2+1"
```

Comment:
```suggestion
    version: "2.10.3"
```

Reply:
not really no? version is the fixed one from agent, this is a revision of the package only, nothing changed upstream in the agent

Reply:
crap, didnt see your comment below loool